geom/CMakeLists.txt

```diff
@@ -15,12 +15,6 @@ add_subdirectory(geompainter)
 
 if(gdml)
   add_subdirectory(gdml)
-  if(NOT gnuinstall)
-    install(DIRECTORY gdml/ DESTINATION geom/gdml
-            FILES_MATCHING PATTERN "*.py"
-            PATTERN "inc" EXCLUDE
-            PATTERN "src" EXCLUDE)
-  endif()
```

Comment:
It looks good in principle, but do we know where these scripts are used, if at all? What purpose do they serve being only available in the build directory? I guess it would only be fine if they were used in some tests alone and they're not supposed to be exposed to users.

Reply:
I don't know. Maybe @agheata has an idea?

Reply:
>  What purpose do they serve being only available in the build directory? 

This is not how I read the `install` invocation.  I read it as installing this file 'only' in the install directory (and not in the build directory) but .... only if not using the `gnuinstall` layout.

> but do we know where these scripts are used, if at all? 

We don't know if they are used.  However according to the doc `geom/gdml/doc/index.md` those are used to convert `TGeo` objects to `gdml` from python.

So if anything it is surprising that they are not installed in all cases.   In a related observation they are not tested by ctest at the moment ... Maybe Witek has some left over tests?

Reply:
I agree with Jonas, the simplification of the build system has more advantages than installing Python scripts that have never been tested and have never been reachable in any ROOT python installation before.